solr/core/src/java/org/apache/solr/handler/admin/PropertiesRequestHandler.java

```diff
@@ -75,4 +76,9 @@ public String getDescription() {
   public Category getCategory() {
     return Category.ADMIN;
   }
+
+  @Override
+  public Name getPermissionName(AuthorizationContext request) {
+    return Name.CONFIG_READ_PERM;
```

Comment:
[-1] The ref-guide describes CONFIG_READ as being scoped mostly to collection level config. (see snippet from the 'Rule Based Authorization Plugin' page below

> this permission is allowed to read a collection’s configuration using the Config API, the Request Parameters API, and other APIs which modify configoverlay.json

IMO widening this definition is totally OK if we don't want to create a new system-level-config perm, but if we're going this route the ref-guide docs should be updated to match.

Reply:
I have extended the documentation for CONFIG_READ to mean read of any solr configuration. Is that ok?